config/routes.rb

```diff
@@ -5,7 +5,9 @@
 
   namespace :api do 
     namespace :v1 do
-      resources :regions, only: [:index, :show]
+      resources :regions, only: [:index, :show] do
+        resources :flavors, only: [:index, :show]
+      end
```

Comment:
this is where we may want to not nest the show action of flavors, and potentially remove the index action since it is not being used

Reply:
Made recommended change